external/anomalib/constraints.txt

```diff
@@ -0,0 +1,20 @@
+albumentations==1.1.0
+attrdict==2.0.1
+defusedxml==0.7.1
+kornia==0.5.6
+lxml==4.6.3
+matplotlib==3.4.3
+networkx==2.5.1
+nncf==2.0.0
```

Comment:
Is it enough to use release version of NNCF?

Reply:
I am not sure. The tests pass so I assume it should be.